src/reference/01-General-Info/05-Using-Sonatype.md

```diff
@@ -203,83 +257,16 @@ range (e.g. Umlauts). If you are absolutely sure that you typed the
 right phrase and the error doesn't disappear, try changing the
 passphrase.
 
-### Summary
-
-To get your project hosted on Sonatype (and Maven Central), you will
-need to:
+### Fourth - Integrate with the release process
 
--   Have GPG key pair, with published public key,
--   An sbt file with your Sonatype credentials *that is not pushed to
-    the VCS*,
--   Add the [sbt-pgp plugin][sbt-pgp] to sign
-    the artefacts,
--   Modify `build.sbt` with the required elements in the generated POM.
-
-Starting with a project that is not being published, you'll need to
-install GPG, generate and publish your key. Swtiching to sbt, you'll
-then need to:
-
-#### ~/.sbt/sonatype.sbt
-
-This file (kept *outside the VCS*) contains the Sonatype credentials
-settings:
-
-```scala
-credentials += Credentials("Sonatype Nexus Repository Manager",
-                           "oss.sonatype.org",
-                           "your-sonatype-username",
-                           "your-sonatype-password")
+To automate the above publishing approach with the [sbt-release plugin]
```

Comment:
Could you mention that sbt-release is a third-party plugin, and that it's not covered by Lightbend subscription.